WARNING 2013-11-21 04:00:23,552 tracking 27225 140313536399104 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:32,715 tracking 27867 140056478889728 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:38,296 tracking 27867 140056132056832 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:42,515 tracking 27867 140056235878144 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:47,530 tracking 27867 140056478889728 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:51,382 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:52,075 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:52,681 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:53,283 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:53,979 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:54,458 jobs 28356 139900156806944 Error in obtaining search results: Bad results from searcher
WARNING 2013-11-21 04:21:54,459 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:54,740 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:55,381 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:55,962 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:56,512 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:57,177 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:57,736 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:58,451 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:59,039 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:21:59,628 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:22:00,313 tracking 28356 139900156806944 ValueError in sh_sess cookie
WARNING 2013-11-21 04:22:00,894 tracking 28356 139900156806944 ValueError in sh_sess cookie
ERROR 2013-11-21 04:22:01,755 speed_logging_utils 28356 139900156806944 browser speed log HTTP request keys missing from JSON object
None
ERROR 2013-11-21 04:22:02,060 speed_logging_utils 28356 139900156806944 browser speed log HTTP request keys missing from JSON object
None
ERROR 2013-11-21 04:22:02,171 speed_logging_utils 28356 139900156806944 browser speed log HTTP request JSON parse failed
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 93, in read_browser_speed_data
    speed_data = json.loads(request.body)
  File "/usr/lib64/python2.7/json/__init__.py", line 326, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 360, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 378, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
ERROR 2013-11-21 04:22:02,349 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:22:02,486 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:22:02,618 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'django_begins'
ERROR 2013-11-21 04:22:02,747 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_begins'
ERROR 2013-11-21 04:22:02,876 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:22:03,012 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (django_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:22:03,150 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:22:03,293 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (php_bridge_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:22:03,430 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_begins'
ERROR 2013-11-21 04:22:03,559 speed_logging_utils 28356 139900156806944 unable to retrieve timestamp from request object for speed logging (php_bridge_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_ends'
WARNING 2013-11-21 04:54:44,812 jobs 29489 140300471658272 Error in obtaining search results: Bad results from searcher
ERROR 2013-11-21 04:54:52,108 speed_logging_utils 29489 140300471658272 browser speed log HTTP request keys missing from JSON object
None
ERROR 2013-11-21 04:54:52,406 speed_logging_utils 29489 140300471658272 browser speed log HTTP request keys missing from JSON object
None
ERROR 2013-11-21 04:54:52,519 speed_logging_utils 29489 140300471658272 browser speed log HTTP request JSON parse failed
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 93, in read_browser_speed_data
    speed_data = json.loads(request.body)
  File "/usr/lib64/python2.7/json/__init__.py", line 326, in loads
    return _default_decoder.decode(s)
  File "/usr/lib64/python2.7/json/decoder.py", line 360, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib64/python2.7/json/decoder.py", line 378, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
ERROR 2013-11-21 04:54:52,705 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:54:52,842 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:54:52,979 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'django_begins'
ERROR 2013-11-21 04:54:53,107 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_begins'
ERROR 2013-11-21 04:54:53,233 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (django_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:54:53,374 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (django_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:54:53,510 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:54:53,646 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (php_bridge_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
AttributeError: 'Request' object has no attribute 'speed'
ERROR 2013-11-21 04:54:53,780 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (php_bridge_begins)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_begins'
ERROR 2013-11-21 04:54:53,909 speed_logging_utils 29489 140300471658272 unable to retrieve timestamp from request object for speed logging (php_bridge_ends)
Traceback (most recent call last):
  File "/home/oleg/code/web-serpng/code/serpng/../serpng/lib/speed_logging_utils.py", line 35, in read
    value = request.speed[name]
KeyError: 'php_bridge_ends'
